x-pack/plugins/monitoring/public/directives/main/index.html

```diff
@@ -1,19 +1,32 @@
 <div class="app-container">
-  <div id="setupModeNav"></div>
-  <kbn-top-nav
-    name="monitoringMain.navName"
-    app-name="'monitoring'"
-    show-search-bar="true"
-    show-auto-refresh-only="!monitoringMain.datePicker.enableTimeFilter"
-    show-date-picker="monitoringMain.datePicker.enableTimeFilter"
-    date-range-from="monitoringMain.datePicker.timeRange.from"
-    date-range-to="monitoringMain.datePicker.timeRange.to"
-    is-refresh-paused="monitoringMain.datePicker.refreshInterval.pause"
-    refresh-interval="monitoringMain.datePicker.refreshInterval.value"
-    on-refresh-change="monitoringMain.datePicker.onRefreshChange"
-    on-query-submit="monitoringMain.datePicker.onTimeUpdate"
-  >
-  </kbn-top-nav>
+  <div class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--justifyContentSpaceBetween euiFlexGroup--directionRow euiFlexGroup--responsive">
+    <div class="euiFlexItem euiFlexItem--flexGrowZero">
+      <div class="euiFlexGroup euiFlexGroup--gutterNone euiFlexGroup--justifyContentSpaceEvenly euiFlexGroup--responsive euiFlexGroup--directionColumn">
+        <div class="euiFlexItem euiFlexItem--flexGrowZero"><div id="setupModeNav"></div></div>
+        <div class="euiFlexItem euiFlexItem--flexGrowZero monTopNavSecondItem">
+          <div data-test-subj="monitoringPageTitle" ng-if="pageTitle || monitoringMain.instance">
+            <h3 class="euiTitle euiTitle--xsmall">{{pageTitle || monitoringMain.instance}}</h3>
```

Comment:
```suggestion
            <h1 class="euiTitle euiTitle--xsmall">{{pageTitle || monitoringMain.instance}}</h1>
```
If this is the H1, we should use the `h1` tag here.